### PR TITLE
docs(docs): unify the style of property type presentation

### PR DESCRIPTION
**Pull request by lxKylin**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

image

### CODE REVIEW

File: docs/en-US/component/menu.md [hunk 1]
```diff
@@ -65,23 +65,23 @@ menu/popper-offset
...
+| popper-effect ^(2.2.26)         | Tooltip theme, built-in theme: `dark` / `light` when menu is collapsed                                                                                                | `string`                | `dark` / `light`          | dark     |
 | close-on-click-outside ^(2.4.4) | optional, whether menu is collapsed when clicking outside                                                                                                             | `boolean`             | —                     | false    |
-| popper-class ^(2.5.0)           | custom class name for all popup menus                                                                                                                                 | string                | —                     | —        |
+| popper-class ^(2.5.0)           | custom class name for all popup menus                                                                                                                                 | `string`                | —                     | —        |
```
Comment (btea) [inline docs/en-US/component/menu.md:84]: The following `sub-menu` attribute style can also be modified uniformly.

File: docs/en-US/component/popover.md [hunk 2]
```diff
@@ -72,26 +72,26 @@ popover/directive-usage
...
+| width                     | popover width                                                                                                                                                            | `string` / `number` | —                                                                                                         | Min width 150px                                                            |
+| placement                 | popover placement                                                                                                                                                        | `string`          | `top`/`top-start`/`top-end`/`bottom`/`bottom-start`/`bottom-end`/`left`/`left-start`/`left-end`/`right`/`right-start`/`right-end` | bottom                                                                     |
```
Comment (btea) [inline docs/en-US/component/popover.md:80]:
```suggestion
| placement                 | popover placement                                                                                                                                                        | `string`          | ^[enum]`'top' \| 'top-start' \| 'top-end' \| 'bottom' \| 'bottom-start' \| 'bottom-end' \| 'left' \| 'left-start' \| 'left-end' \| 'right' \| 'right-start' \| 'right-end' \| 'bottom'`                                                                     |
```

File: docs/en-US/component/popover.md [hunk 3]
```diff
@@ -72,26 +72,26 @@ popover/directive-usage
...
-| persistent                | when popover inactive and `persistent` is `false` , popover will be destroyed                                                                                            | boolean         | —                                                                                                         | true                                                                       |
+| trigger                   | how the popover is triggered                                                                                                                                             | `string`          | `click/focus/hover/contextmenu`                                                                             | hover                                                                      |
```
Comment (btea) [inline docs/en-US/component/popover.md:75]:
```suggestion
| trigger                   | how the popover is triggered                                                                                                                                             | `string`          | ^[enum]`'click' \| 'focus' \| 'hover' \| 'contextmenu'`                                                                             | hover                                                                      |
```

File: docs/en-US/component/datetime-picker.md [hunk 4]
```diff
@@ -77,34 +77,34 @@ datetime-picker/default-time
...
+| start-placeholder       | placeholder for the start date in range mode                                                                   | ^[string]                                           | —                                                             | —                   |
+| end-placeholder         | placeholder for the end date in range mode                                                                     | ^[string]                                           | —                                                             | —                   |
+| arrow-control           | whether to pick time using arrow buttons                                                                       | ^[boolean]                                          | —                                                             | false               |
+| type                    | type of the picker                                                                                             | ^[string]                                           | ^[year]/^[month]/^[date]/^[datetime]/ ^[week]/^[datetimerange]/^[daterange]        | date                |
```
Comment (btea) [inline docs/en-US/component/datetime-picker.md:90]: In cases like this where there are multiple enumeration values, we might be able to use `enum`.

File: docs/en-US/component/cascader.md [hunk 5]
```diff
@@ -161,7 +161,7 @@ cascader/panel
...
 | max-collapse-tags ^(2.3.10)         | The max tags number to be shown. To use this, `collpase-tags` must be true                                                                                                       | ^[number]                                                     | 1       |
 | empty-values ^(2.7.0)               | empty values of component, [see config-provider](/en-US/component/config-provider#empty-values-configurations)                                                                   | ^[array]                                                      | —       |
-| value-on-clear ^(2.7.0)             | clear return value, [see config-provider](/en-US/component/config-provider#empty-values-configurations)                                                                          | ^[string] / ^[number] / ^[boolean] / ^[Function]              | —       |
+| value-on-clear ^(2.7.0)             | clear return value, [see config-provider](/en-US/component/config-provider#empty-values-configurations)                                                                          | ^[enum]`'string'\|'number'\|'boolean'\|'Function'`              | —       |
```
Comment (tolking) [inline docs/en-US/component/cascader.md:164]: The `^[string] / ^[number] / ^[boolean] / ^[Function]` is not an enum type.

The correct enumeration type is like this `^[enum]`'large'\| 'default'\| 'small'``

File: docs/en-US/component/carousel.md [hunk 6]
```diff
@@ -79,18 +79,18 @@ carousel/vertical
 
 | Name                 | Description                                           | Type    | Accepted Values     | Default    |
```
Comment (tolking) [inline docs/en-US/component/carousel.md:80]: ~~Accepted Values~~

The new version of the document has removed the Accepted Values property

Comment (lxKylin) [reply docs/en-US/component/carousel.md:80]: I look at the latest code, this is not deleted
image

**Comment by tolking** [comment]
Thank for your contribution.

I noticed that wzc520pyfm has already made relevant modifications. 

Link #10754

**Pull request closed without merging**